enterprise/app/controllers/enterprise/api/v1/accounts_controller.rb

```diff
@@ -10,12 +12,40 @@ def subscription
     head :no_content
   end
 
+  def limits
+    limits = {
+      'conversation' => {},
+      'non_web_inboxes' => {}
+    }
+
+    if default_plan?(@account)
+      limits = {
+        'conversation' => {
+          'allowed' => 500,
+          'consumed' => conversations_this_month(@account)
+        },
+        'non_web_inboxes' => {
+          'allowed' => 0,
+          'consumed' => non_web_inboxes(@account)
+        }
+      }
+    end
+
+    # include id in response to ensure that the store can be updated on the frontend
+    render json: { id: @account.id, limits: limits }, status: :ok
+  end
+
   def checkout
     return create_stripe_billing_session(stripe_customer_id) if stripe_customer_id.present?
 
     render_invalid_billing_details
   end
 
+  def check_cloud_env
+    installation_config = InstallationConfig.find_by(name: 'DEPLOYMENT_ENV')
```

Comment:
@scmmishra You were picking this value from ENV. It should have been taken from the installation config.